localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_human_1;Knightly Stables;;Ritterliche Stallungen;;;;;;;;;;;x
+ca_culture_human_2;Knightly Stables;;Ritterliche Stallungen;;;;;;;;;;;x
+ca_culture_human_3;Knightly Stables;;Ritterliche Stallungen;;;;;;;;;;;x
+ca_culture_human_4;Knightly Stables;;Ritterliche Stallungen;;;;;;;;;;;x
+ca_culture_human_desc;Knights are traditionally powerful, horse-mounted warriors based in the human kingdoms. Knighthood among humankind has became an incredible symbol of honour and respect. These mounted warriors wield weapons of their choice, but are most often found with lance or sword.;;Ritter sind traditionell m�chtige, auf Pferden reitende Krieger, die in den Menschenk�nigreichen beheimatet sind. Das Rittertum ist unter den Menschen zu einem eindrucksvollen Symbol f�r Ehre und Respekt geworden. Diese berittenen Krieger f�hren Waffen ihrer Wahl, am h�ufigsten trifft man sie jedoch mit Lanze oder Schwert an.;;;;;;;;;;;x
+ca_culture_naga_1;Myrmidon Encampment;;Myrmidonen-Lager;;;;;;;;;;;x
+ca_culture_naga_2;Myrmidon Encampment;;Myrmidonen-Lager;;;;;;;;;;;x
+ca_culture_naga_3;Myrmidon Encampment;;Myrmidonen-Lager;;;;;;;;;;;x
+ca_culture_naga_4;Myrmidon Encampment;;Myrmidonen-Lager;;;;;;;;;;;x
+ca_culture_naga_desc;The Royal Guard are elite naga warriors sworn to protect the utmost important of individuals in naga society. Mostly naga males get to serve in this special force, as they are among the most physically and magically talented amongst their brethren. Due to their physical strength and magical prowess, they have the capability to turn the tides of the battle, lunging trident-first into the lines of combatants.;;Die K�nigliche Garde sind Elite-Naga-Krieger, die geschworen haben, die bedeutendsten Pers�nlichkeiten der Naga-Gesellschaft zu sch�tzen. Haupts�chlich m�nnliche Naga d�rfen in dieser Spezialeinheit dienen, wenn sie zu den k�rperlich und magisch talentiertesten ihrer Br�der geh�ren. Aufgrund ihrer k�rperlichen St�rke und ihrer magischen F�higkeiten sind sie in der Lage, die Geschicke der Schlacht zu lenken, indem sie sich mit dem Dreizack voran in die Reihen der K�mpfer st�rzen.;;;;;;;;;;;x
```

Comment:
Utmost did not make sense in that context.
```suggestion
ca_culture_naga_desc;The Royal Guard are elite naga warriors sworn to protect the most important individuals in naga society. Mostly naga males get to serve in this special force, as they are among the most physically and magically talented amongst their brethren. Due to their physical strength and magical prowess, they have the capability to turn the tides of the battle, lunging trident-first into the lines of combatants.;;Die K�nigliche Garde sind Elite-Naga-Krieger, die geschworen haben, die bedeutendsten Pers�nlichkeiten der Naga-Gesellschaft zu sch�tzen. Haupts�chlich m�nnliche Naga d�rfen in dieser Spezialeinheit dienen, wenn sie zu den k�rperlich und magisch talentiertesten ihrer Br�der geh�ren. Aufgrund ihrer k�rperlichen St�rke und ihrer magischen F�higkeiten sind sie in der Lage, die Geschicke der Schlacht zu lenken, indem sie sich mit dem Dreizack voran in die Reihen der K�mpfer st�rzen.;;;;;;;;;;;x
```

Reply:
agreed